src/main/resources/marc4bibframeAuthority.yml

```diff
@@ -0,0 +1,70 @@
+authorityFieldRules:
+  '100':
+    - types: CONCEPT, PERSON
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '!3'
+        fieldsOneOf:
+          v: presented
+          x: presented
+          y: presented
+          z: presented
+      edges:
+        - types: PERSON
+          predicate: FOCUS
+          subfields:
+            a: NAME
+            b: NUMERATION
+            c: TITLES
+            d: DATE
+            j: ATTRIBUTION
+            q: NAME_ALTERNATIVE
+    - types: CONCEPT, FAMILY
+      include: 100_mappings
+      marc2ldCondition:
+        ind1: '3'
```

Comment:
fieldsOneOf:
          v: presented
          x: presented
          y: presented
          z: presented
          
 is needed here also.

Reply:
Thanks, add test cases